File: src/test/java/br/com/newObjects/model/entities/classes/NovoTelefone.java
package br.com.newObjects.model.entities.classes;

import br.com.model.entities.classes.Telefone;
import br.com.model.entities.classes.usuario.Usuario;

public class NovoTelefone {

    public static void novaListaTelefones(Usuario usuario) {
        new Telefone(usuario, "(ii)thhhh-hhhh");
        new Telefone(usuario, "(yy)qaaaa-zzzz");
    }

    public static void novoTelefone(Usuario usuario) {
        new Telefone(usuario, "(xx)yyyy-yyyy");

    }
}

File: src/test/java/br/com/newObjects/model/entities/classes/NovoTelefone.kt
package br.com.newObjects.model.entities.classes

import br.com.model.entities.classes.Telefone
import br.com.model.entities.classes.usuario.Usuario

object NovoTelefone {
    @JvmStatic
    fun novaListaTelefones(usuario: Usuario) {
        Telefone(usuario, "(ii)thhhh-hhhh")
        Telefone(usuario, "(yy)qaaaa-zzzz")
    }

    @JvmStatic
    fun novoTelefone(usuario: Usuario) {
        Telefone(usuario, "(xx)yyyy-yyyy")
    }
}
